lib/Common/Memory/RecyclerPointers.h

```diff
@@ -489,21 +489,26 @@ template <class Policy>
 struct _QuickSortImpl
 {
     template<class T, class Comparer>
-    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context)
+    static void Sort(T* arr, size_t count, const Comparer& comparer, void* context, size_t elementSize = sizeof(T))
     {
+#ifndef _WIN32
+        JsUtil::QuickSort<Policy, char, Comparer>::Sort((char*)arr, count, elementSize, comparer, context);
```

Comment:
On XPLAT, this implementation replaces qsort_s. Are we sure all the current and future usages of this will comply with what you ask? In other words, T could be the type that `=` operator of it wouldn't perform what we want. 

This implementation, once the type is no_write_barrier_policy, it copies the memory.